File: masterloop.cloud.portal/src/app/core/models/tenant.ts
import { AddOnFeature } from '../enums/add-on-feature.enum';

export class Tenant {
  id: number;
  name: string;
  features: AddOnFeature[];
  templateIds: string[];

  constructor(
    id: number,
    name: string,
    features: AddOnFeature[],
    templateIds: string[]
  ) {
    this.id = id;
    this.name = name;
    this.features = features;
    this.templateIds = templateIds;
  }
}

File: masterloop.cloud.api/src/Masterloop.Cloud.Core/Tenant/Tenant.cs
namespace Masterloop.Cloud.Core.Tenant
{
    public class Tenant
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public AddOnFeature[] Features { get; set; }

        public string[] TemplateIds { get; set; }
    }
}
